app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class RentMailer < ApplicationMailer
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.rent_mailer.send.subject
+  #
+  def new_rent(email, first_name, title)
```

Comment:
Here you forgot to add the creation date and the from and to from the rent to the email data.

Reply:
I didn't. But Rubocop didn't let me push because it was too much. And if I put the complete hash, the worker fails.